lib-injection/build/docker/java/dd-djm-spark-test-app/launch.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Submit a example spark job with DJM enabled
+$SPARK_HOME/bin/spark-submit   \
+--class org.apache.spark.examples.SparkPi  \
+ --master k8s://https://$KUBERNETES_SERVICE_HOST:$KUBERNETES_SERVICE_PORT   \
+ --conf spark.kubernetes.container.image=apache/spark:3.4.4  \
+ --deploy-mode cluster  \
+ --conf spark.kubernetes.namespace=default   \
+ --conf spark.kubernetes.executor.deleteOnTermination=false   \
+ --conf spark.kubernetes.driver.label.admission.datadoghq.com/enabled=true   \
+ --conf spark.kubernetes.authenticate.driver.serviceAccountName=spark   \
+ --conf spark.kubernetes.authenticate.executor.serviceAccountName=spark   \
+ --conf spark.kubernetes.driver.annotation.admission.datadoghq.com/java-lib.version=latest   \
```

Comment:
Instead of using annotation `admission.datadoghq.com/java-lib.version`, use `admission.datadoghq.com/java-lib.custom-image` as the value is readily available in the `test_k8s_instance` test fixture.